fluent-plugin-winevtlog.gemspec

```
@@ -20,7 +20,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "bundler"
   spec.add_development_dependency "rake"
   spec.add_development_dependency "test-unit", "~> 3.2.0"
-  spec.add_development_dependency "nokogiri", [">= 1.10", "< 1.12"]
+  spec.add_development_dependency "nokogiri", [">= 1.10", "~> 1.11.pre"]
```

How about this?
The following change should be succeeded to install nokogiri 1.11.0.rc2.
```suggestion
  spec.add_development_dependency "nokogiri", [">= 1.11.pre", "< 1.12"]
```

---

It seems that it is reasonable change.